pkgs/top-level/python-packages.nix

```diff
@@ -7574,6 +7574,21 @@ let
     propagatedBuildInputs = with self; [ eventlib application gnutls ];
   };
 
+  multipledispatch = buildPythonPackage rec {
+    name = "multipledispatch-${version}";
+    version = "0.4.8";
+   
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/m/multipledispatch/${name}.tar.gz";
+      md5 = "f0eb0c527cdbe788ca7dcd1511bc49a9";
+    };
+    
+    meta = {
+      homepage = "http://github.com/mrocklin/multipledispatch/";
+      description = "A relatively sane approach to multiple dispatch in Python.";
```

Comment:
Descriptions should not end with a dot. See http://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes